src/lime/_internal/backend/html5/HTML5Application.hx

```diff
@@ -343,6 +361,17 @@ class HTML5Application
 		return 0;
 	}
 
+	public function checkFlagHidden():Bool 
```

Comment:
If this function is going to be public, we need to think carefully about the name because it's going to become a (semi)permanent part of the API.

I don't think "checkFlagHidden" does a good job of describing the function. Someone only looking at the name wouldn't know which flag you were talking about. "checkDocumentHidden" lets you know what is (or isn't) hidden: the HTML document.

I'm also not a fan of "check", mainly because Lime doesn't normally use the word in that way. Based on how Lime uses words, the most appropriate wording would be "getDocumentHidden". Either that or a property with a getter: "`var documentHidden(get, never)`".